src/clusterfuzz/_internal/bot/tasks/utasks/progression_task.py

```diff
@@ -285,23 +389,33 @@ def find_fixed_range(testcase_id, job_type):
 
   min_index = revisions.find_min_revision_index(revision_list, min_revision)
   if min_index is None:
-    raise errors.BuildNotFoundError(min_revision, job_type)
+    error_message = f'Build {min_revision} no longer exists'
+    return uworker_io.UworkerOutput(
+          testcase=testcase,
+          error_message=error_message,
+          error=uworker_msg_pb2.ErrorType.PROGRESSION_BUILD_NOT_FOUND)      
   max_index = revisions.find_max_revision_index(revision_list, max_revision)
   if max_index is None:
-    raise errors.BuildNotFoundError(max_revision, job_type)
+    error_message = f'Build {max_revision} no longer exists'
+    return uworker_io.UworkerOutput(
+          testcase=testcase,
+          error_message=error_message,
+          error=uworker_msg_pb2.ErrorType.PROGRESSION_BUILD_NOT_FOUND)
 
   testcase = data_handler.get_testcase_by_id(testcase_id)
-  data_handler.update_testcase_comment(testcase, data_types.TaskState.STARTED,
-                                       'r%d' % max_revision)
 
   # Check to see if this testcase is still crashing now. If it is, then just
   # bail out.
-  result = _testcase_reproduces_in_revision(
+  result, error = _testcase_reproduces_in_revision(
       testcase,
       testcase_file_path,
       job_type,
       max_revision,
+      # Changed by me to false
```

Comment:
What does this comment mean?
Also nit: End comments with a period please.

Reply:
You can ignore this, left here by mistake.